### DISCUSSION TITLE
'boost::placeholders' has not been declared

### DISCUSSION THREAD

**Discussion by ikamran** [dogecoin/dogecoin#2941] [Help needed]
During compiling I got these errors:

Making all in src
make[1]: Entering directory '/root/dogecoin/src'
make[2]: Entering directory '/root/dogecoin/src'
make[3]: Entering directory '/root/dogecoin'
make[3]: Leaving directory '/root/dogecoin'
make[3]: Entering directory '/root/dogecoin/src/secp256k1'
make[3]: Leaving directory '/root/dogecoin/src/secp256k1'
  CXX      rpc/libdogecoin_server_a-server.o
rpc/server.cpp: In function 'void RPCServer::OnPreCommand(boost::function<void(const CRPCCommand&)>)':
rpc/server.cpp:61:62: error: 'boost::placeholders' has not been declared
     g_rpcSignals.PreCommand.connect(boost::bind(slot, boost::placeholders::_1));
                                                              ^
rpc/server.cpp: In function 'void RPCServer::OnPostCommand(boost::function<void(const CRPCCommand&)>)':
rpc/server.cpp:66:63: error: 'boost::placeholders' has not been declared
     g_rpcSignals.PostCommand.connect(boost::bind(slot, boost::placeholders::_1));
                                                               ^
rpc/server.cpp: In member function 'std::vector<std::__cxx11::basic_string<char> > CRPCTable::listCommands() const':
rpc/server.cpp:520:39: error: 'boost::placeholders' has not been declared
                                boost::placeholders::_1) );
                                       ^
Makefile:5939: recipe for target 'rpc/libdogecoin_server_a-server.o' failed
make[2]: *** [rpc/libdogecoin_server_a-server.o] Error 1
make[2]: Leaving directory '/root/dogecoin/src'
Makefile:9521: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/root/dogecoin/src'
Makefile:687: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1

what is the issue, I installed all dependencies

**Comment by patricklodder** [comment]
This means your boost is out of date, sorry. You need at least boost 1.60. [We fixed this bug in our autoconf for 1.14.6](https://github.com/dogecoin/dogecoin/pull/2693)
